src/pages/Categories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useDispatch } from 'react-redux/es/exports';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+  const CHECK_STATUS = 'book-store/categories/CHECK_STATUS';
+  return (
+    <>
+      <button type="button" onClick={() => dispatch({ type: CHECK_STATUS })}>
```

Comment:
Instead of creating the `CHECK_STATUS` action type again here, by having the action creator in the categories reducer you get to use the variable that you've already created.

- Kindly remove the repetition of code here and simply import the `checkStatus` action creator and dispatch it in the `onClick` event instead.